tests/software/metaphlan3/run/main.nf

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { UNTAR          } from '../../../../software/untar/main.nf'          addParams( options: [:] )
+include { SAMTOOLS_VIEW  } from '../../../../software/samtools/view/main.nf'  addParams( options: ['suffix': '.sam'] ) //metaphlan can't take bam input
+include { METAPHLAN3_RUN } from '../../../../software/metaphlan3/run/main.nf' addParams( options: [ 'args':'--index mpa_v30_CHOCOPhlAn_201901 --add_viruses --bt2_ps very-sensitive-local' ] ) //database version (latest, if not supplied attempts database autodownload and run fails)
+
+workflow test_metaphlan3_single_end {
+
+    input = [ [ id:'test', single_end:true ], // meta map
+              [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true) ]
+            ]
+
+    db    = channel.fromPath('https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/delete_me/metaphlan_database.tar.gz', type: 'dir', checkIfExists: true)
```

Comment:
Fine for now, but since this looks like a good test dataset, we can actually add this to the `genomics` folder I'd say. Maybe something to discuss on slack :)

Reply:
I think the metaphlan marker database I created may be better suited as a draft at the moment as it needs a little work.. It seems to be 'working' fine (input successfully aligned and output is generated) but it struggles to fully classify the test data (tool authors suggested this could be related to low coverage of input data when I asked about this..) I think it is ok at the moment for testing but I hope to improve it (great shout; I'm sure the nf-core folks will have some good ideas on building custom metaphlan DB!)